gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/launcher/ProcessWorkUnitsJobLauncher.java

```diff
@@ -93,4 +97,11 @@ public void submitJob(List<WorkUnit> workunits) {
       throw new RuntimeException(e);
     }
   }
+
+  private void setupMDCFromJobState(JobState jobState) {
+    // TODO: log4j2 has better syntax around conditional logging such that the key does not need to be included in the value
```

Comment:
the key may actually assist machine processing.  I'd suggest just changing the key names, so they're actually ready to be fed directly back into the GaaS REST endpoints, such as:
```
GET /flowexecutions/(flowGroup:<groupName>,flowName:<flowName>,flowExecutionId:<flowExecutionId>)
```
hence `flowName`, rather than `flow.name`.  bonus points if these can come out comma-separated, and so ready to use w/ that REST API, rather than space-separated as you showed in the example output.

Reply:
Yeah this is a good point, it might make it harder to set up queries on logs though if we keep the keys in the values. I'll change the keys to match what the API accepts and format the logs appropriately.